tools/db_bench_tool.cc

```diff
@@ -528,6 +524,8 @@ DEFINE_bool(new_table_reader_for_compaction_inputs, true,
 
 DEFINE_int32(compaction_readahead_size, 0, "Compaction readahead size");
 
+DEFINE_int32(table_evict_type, 0, "Table evict type");
```

Comment:
Maybe we should provide a description of the potential options here

Reply:
updated